### PR TITLE
Add custom API to load orders

### PR DESCRIPTION
**Pull request by Agusx1211**
Adds a custom UniswapEX API to load the orders of the user, the new API is used alongside with the old Etherscan method, in parallel

### CODE REVIEW

**Comment by nachomazzara** [comment]
Nice work Agus! What do you think of adding a check to remove cancelled orders when fetching orders by the API?

**Comment by Agusx1211** [comment]
That check already exists, when an order get's cancelled the amount of the order becomes zero, and it's not rendered

**Comment by nachomazzara** [comment]
I canceled an order and it has not refreshed. Where is the check? Maybe we need to enforce something

File: webapp/src/components/ExchangePage/index.jsx [hunk 1]
```diff
@@ -369,7 +369,97 @@ function getMarketRate(
...
+      ])
+      const fetchedOrders = await requestOrders.json()
+      for (const order of fetchedOrders) {
+        // Get balance of the order
+        order.fee = ethers.utils.bigNumberify(order.fee.toString())
+        order.minReturn = ethers.utils.bigNumberify(order.minReturn.toString())
+
+        const amount = await amountOfOrder(order, uniswapEXContract)
+        const key = keyOfOrder(order)
+        if (!amount.isZero() && ordersAdded[key] === undefined) {
```
Comment (nachomazzara) [inline webapp/src/components/ExchangePage/index.jsx]: Maybe this is preventing from doing the update of amount 0?

**Comment by Agusx1211** [comment]
Replaced by #53

**Pull request closed without merging**